src/pages/home/uploadSong/AdvancedSongDetails.tsx

```diff
@@ -129,12 +143,54 @@ const AdvancedSongDetails = () => {
           }
           max={ new Date().toISOString().split("T")[0] }
         />
-        <TextInputField
-          name="copyright"
-          label="COPYRIGHT"
-          placeholder={ `${new Date().getFullYear()} Example` }
-          ref={ copyrightRef }
-          tooltipText={ "" }
+        <CopyrightInputField
+          ref={ phonographicCopyrightRef }
+          label="SOUND RECORDING COPYRIGHT"
+          yearFieldName="phonographicCopyrightYear"
+          ownerFieldName="phonographicCopyrightOwner"
+          copyrightType="phonographic"
+          isOptional={ false }
+          tooltipText={
+            <span>
+              The copyright in a sound recording covers the recording itself (it
+              does not cover the music or lyrics of the song). It is typically
+              owned by the artist and/or record label. If you are not the
+              copyright holder of the sound recording, please review{ " " }
+              <Link
+                href="https://newm.io/artists-faq"
```

Comment:
1. Created a `constants.ts` file so we don't have to jump across multiple files for external link changes
2. This link needs to be updated per one of the go-live-testing bug proposals, will be a quick change as part of that bug once the url is identified.
3. Might need to update to `NEWM_STUDIO` or just remove the `ARTISIT_PORTAL` portion of the naming convention completely if not relevant

https://github.com/projectNEWM/newm-artist-portal/blob/c62f1a9929ba131c3f14a41e3d0822b346eda832/src/common/constants.ts#L11